fastapi_tut/api/api_v1/endpoints/user.py

```diff
@@ -0,0 +1,102 @@
+# initial, todo
+
+from sqlmodel import Session
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi_jwt_auth import AuthJWT
+from typing import List
+
+from fastapi_tut import crud
+from fastapi_tut.api import deps
+from fastapi_tut.models.user import User, UserCreate, ShowUser, UserUpdate
+
+router = APIRouter()
+
+@router.get("/profile", response_model=ShowUser)
+async def read_profile(
+    db : Session = Depends(deps.get_db),
+    current_user: ShowUser = Depends(deps.get_current_user)
+):
+    
+    return current_user
+
+# Module 3: Exam
+@router.get("/", response_model=List[User])
+async def read_users(
+    db : Session = Depends(deps.get_db),
+    current_user: ShowUser = Depends(deps.get_current_user)
+):
+    
+    if not crud.user.is_superuser(current_user):
+        raise HTTPException(status_code=400, details='Not enough permissions.')
+
+    users = crud.user.get_users(db)
```

Comment:
sa `/crud/base.py`
- may get_multi method na ang parent class `CRUDBase` w/ offset and limit options